CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
-cmake_minimum_required(VERSION 2.8.12)
-# When using CMake 3.4 and later, don't export symbols from executables unless
+cmake_minimum_required(VERSION 3.9)
```

Comment:
I bumped this requirement to use `GENERATOR_IS_MULTI_CONFIG` but I understand that it can be an issue; this property can easily be emulated by testing `CMAKE_GENERATOR` instead, but then again I saw that this `CMakeLists` was actually already using some features that are only available in CMake 3 (such as `TARGET_OBJECTS` for instance), so I wasn't sure what was the "true" minimum requirement.